applications/acdc/src/acdc_agent_fsm.erl

```diff
@@ -936,10 +936,10 @@ answered({'sync_req', JObj}, #state{agent_listener=AgentListener
     {'next_state', 'answered', State};
 answered({'channel_unbridged', CallId}, #state{member_call_id=CallId}=State) ->
     lager:info("caller channel ~s unbridged", [CallId]),
-    {'next_state', 'wrapup', State#state{wrapup_ref=wrapup_timer(State)}};
+    {'next_state', 'answered', State};
 answered({'channel_unbridged', CallId}, #state{agent_call_id=CallId}=State) ->
     lager:info("agent channel unbridged"),
-    {'next_state', 'wrapup', State#state{wrapup_ref=hangup_call(State)}};
+    {'next_state', 'answered', State};
```

Comment:
@danielfinke 
Ah, ok that makes sense.  I am not 100% sure of the impact I will need some time to follow the code path.  I suspect it is for offhook agents.  

Once I can verify that it doesn't impact existing functionality the rest of the PR looks good.